back-end/API/app/controllers/enterprise_controller.py

```
@@ -88,8 +93,8 @@ def getCurrentUserEnterprise(current_user):
         if enterprise:
             return jsonify(enterprise.to_json_string()), 200
         else:
-            logger.warn("No enterprise found the current user")
+            logger.warning("No enterprise found the current user")
             return jsonify({'message': 'No enterprise found on the current user'}), 404
     except Exception as e:
-        logger.error("Error getting the user to get the enterprise", e)
+        logger.exception("Error getting the user to get the enterprise" + str(e))
```

Ruff (0.15.2)

[warning] Logging statement uses `+`

(G003)